lib/league_statistics.rb

```diff
@@ -0,0 +1,100 @@
+require_relative "./stat_tracker"
+require "csv"
+
+class LeagueStatistics < StatTracker
+
+  def count_of_teams
```

Comment:
count of teams is asking for the total count/how many teams overall that we're dealing with. This method uses CSV.read to get an array of all the rows in the team file and that is all the teams playing